conformance-checking/src/main/java/org/archcnl/conformancechecking/impl/ConformanceCheckOntologyClassesAndProperties.java

```diff
@@ -98,4 +64,59 @@ public String getUri() {
                     + this.name();
         }
     }
+
+    public static Individual createIndividual(ConformanceCheckOntClasses clazz, OntModel model) {
+        return clazz.createIndividual(model);
+    }
+
+    public static Individual createIndividual(
+            ConformanceCheckOntClasses clazz, OntModel model, int id) {
+        return clazz.createIndividual(model, Integer.toString(id));
+    }
+
+    public enum ConformanceCheckOntClasses {
+        ConformanceCheck,
+        ArchitectureRule,
+        ArchitectureViolation,
+        Proof,
+        AssertedStatement,
+        NotInferredStatement;
+
+        public Individual createIndividual(OntModel model) {
+            String id = getId();
+            return createIndividual(model, id);
+        }
+
+        public Individual createIndividual(OntModel model, String id) {
+            return model.getOntClass(getUri()).createIndividual(getUri() + id);
+        }
+
+        public String getUri() {
+            return ConformanceCheckOntologyClassesAndProperties.getOntologyNamespace()
+                    + this.name();
+        }
+
+        public String getId() {
```

Comment:
Going by the previous comment this method should also return an int (returning null in the default case should be fine). This would also shorten the code a little. If possible I would try to avoid using switch statements but since I also did not come up with a better solution, it is fine. You could also add returns for each case to avoid the breaks but that is a matter of preference. Last but not least it should be private not public as it is currently not accessed outside the enum.